benchexec/tablegenerator/react-table/src/components/QuantilePlot.js

```diff
@@ -205,7 +212,7 @@ export default class QuantilePlot extends React.Component {
           const task = this.state.selection;
           const data = this[task + i];
           const id = getRunSetName(tool);
-          this.lineCount++;
+          this.lineCount += 1;
```

Comment:
What is the purpose of changes like this?

Reply:
That change seems to have happened because of this eslint rule:
eslint.org/docs/rules/no-plusplus